src/main/webapp/courseLoader.js

```diff
@@ -1,127 +1,200 @@
-    diffChart.draw(diffData, diffOptions);
-
-
-    const tempTermPerceptionList = dataObject.termPerceptionList;
-    const termPerceptionList = [["Term Perception"], /* dummyPerceptionRating */ [11], /* dummyPerceptionRating */ [5]].concat(tempTermPerceptionList);
-    const termPerceptionData = new google.visualization.arrayToDataTable(termPerceptionList);
-    const termPerceptionOptions = {
-        title: "Perception of Term Reviews",
-        legend: {position: "none"},
-        vAxis: {title: "Perception"},
-        hAxis: {title: "Comment Quantity"},
-        histogram: {
-          hideBucketItems: true
-       }
-    };
-    const termPerceptionChart = new google.visualization.Histogram(document.getElementById("term-chart"));
-    termPerceptionChart.draw(termPerceptionData, termPerceptionOptions);
-
-
-    const tempProfPerceptionList = dataObject.termPerceptionList;
-    const profPerceptionList = [["Professor Perception"], /* dummyPerceptionRating */ [2], /* dummyPerceptionRating */ [9]].concat(tempProfPerceptionList);
-    const profPerceptionData = new google.visualization.arrayToDataTable(profPerceptionList);
-    const profPerceptionOptions = {
-        title: "Perception of Professor Reviews",
-        legend: {position: "none"},
-        vAxis: {title: "Perception"},
-        hAxis: {title: "Comment Quantity"},
-        histogram: {
-          hideBucketItems: true
-        }
-      };
-    const profPerceptionChart = new google.visualization.Histogram(document.getElementById("prof-chart"));
-    profPerceptionChart.draw(profPerceptionData, profPerceptionOptions);
+document.addEventListener("DOMContentLoaded", function () {
+  populateData();
+});
+
+function createComment(commentText, flag) {
+  const commentContainer = flag
+    ? document.getElementById("course-comments")
+    : document.getElementById("prof-comments");
+
+  const commentWrapper = document.createElement("div");
+  commentWrapper.setAttribute("class", "media text-muted pt-3");
+
+  const commentUser = document.createElement("strong");
+  commentUser.setAttribute("class", "d-block text-gray-dark");
+  commentUser.innerHTML = "@username";
+
+  const commentBody = document.createElement("p");
+  commentBody.setAttribute(
+    "class",
+    "media-body pb-3 mb-0 small lh-125 border-bottom border-gray"
+  );
+  commentBody.innerHTML = commentText;
+
+  commentBody.insertAdjacentElement("afterbegin", commentUser);
+  commentWrapper.appendChild(commentBody);
+  commentContainer.appendChild(commentWrapper);
 }
 
+function loadComments(commentList, flag) {
+  for (let comment of commentList) {
+    createComment(comment, flag);
+  }
+}
+
+function makeGraphs(dataObject) {
+  const termCommentsList = dataObject.termCommentsList;
+  const dummyComments = ["dummy comment 1", "dummy comment 2"].concat(
+    termCommentsList
+  );
+  loadComments(dummyComments, /* true means course comments */ true);
```

Comment:
The comments here on true/false are good. Even better would be a more descriptive flag "isCourse" and just putting that name next to the boolean here.